src/initialization/MOM_state_initialization.F90

```diff
@@ -1999,11 +2007,11 @@ subroutine set_Flather_Bdry_Conds(OBC, tv, h, G, PF, tracer_Reg)
     enddo ; enddo ; enddo
   endif
 
-  do k=1,nz ; do j=js,je ; do I=is-1,ie
+  do k=1,nz ; do j=js-1,je+1 ; do I=is-1,ie+1
```

Comment:
@Hallberg-NOAA  and I were just discussing the extension of this loop range. If it needs to be extended at all then we think it would be to
```
do k=1,nz ; do j=js-1,je+1 ; do I=is-2,ie+1
```
based purely on symmetry.

Can you remind us why you think this loop extension is necessary?

I will add that we're wondering if all the loops preceding this should also be extended to the entire data domain (isd:ied, ...) and that we've survived with the current loop ranges solely because we always had the OB on the processor boundaries.